web/lib/handlers/order.js

```diff
@@ -27,6 +27,21 @@ const post = params => // eslint-disable-line no-unused-vars
       reply(req.app.order).code(200);
       return next();
     },
+  ].concat(sendBookingEmail));
```

Comment:
Super minor, but I'd rather destructuring here.
```
      reply(req.app.order).code(200);
      return next();
    },
    ...sendBookingEmail,
  ]
```
I think it just looks cleaner and a bit more obvious.

Reply:
oky :]